app/services/concerns/xccdf/rule_group_relationships.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving RuleGroupRelationships
+  module RuleGroupRelationships
+    extend ActiveSupport::Concern
+    included do
+      def save_rule_group_relationships
+        @op_rules_and_rule_groups = @op_rule_groups + @op_rules
+
+        ::RuleGroupRelationship.import!(
+          rule_group_relationships.select(&:new_record?), ignore: true
+        )
+
+        rgr_links_to_remove(::RuleGroupRelationship.all).delete_all
+      end
+
+      private
+
+      def rule_group_relationships
+        @rule_group_relationships ||= @op_rules_and_rule_groups.flat_map do |op_r_or_rg|
```

Comment:
Why are you setting this variable if you never use this.

Reply:
That's true.. It's not necessary. I'll remove it!